typing/includecore.ml

```diff
@@ -198,24 +201,24 @@ let report_constructor_mismatch first second decl ppf err =
   | Explicit_return_type ord ->
       pr "%s has explicit return type and %s doesn't."
         (String.capitalize_ascii (choose ord first second))
-        (choose (Ctype.Unification_trace.swap_position ord) first second)
+        (choose_other ord first second)
 
 let report_variant_mismatch first second decl ppf err =
   let pr fmt = Format.fprintf ppf fmt in
   match (err : variant_mismatch) with
   | Constructor_mismatch err ->
     report_constructor_mismatch first second decl ppf err
   | Constructor_names (n, name1, name2) ->
-      pr "Fields number %i have different names, %s and %s."
+      pr "Constructors number %i have different names, %s and %s."
         n (Ident.name name1) (Ident.name name2)
   | Constructor_missing (ord, s) ->
-      pr "The field %s is only present in %s %s."
+      pr "The constructor %s is only present in %s %s."
         (Ident.name s) (choose ord first second) decl
 
 let report_extension_constructor_mismatch first second decl ppf err =
   let pr fmt = Format.fprintf ppf fmt in
   match (err : extension_constructor_mismatch) with
-  | Privacy -> pr "A private type would be revealed"
+  | Constructor_privacy -> pr ".A private type would be revealed"
```

Comment:
Why is there a new dot at the beginning of the message here? (Just a typo?)

Reply:
Yes, it's a typo.